### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by joyhughes**
README file suitable for students in GSSoC '24

### CODE REVIEW

File: Lux/README.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ For an overview of the development history of the project see the [Jen README](h
 Via Emscripten, C++ and JavaScript can have access to shared memory space. This allows the ImagePortCanvas to be rendered in 1-2 milliseconds once the image is rendered.
 
 ## API
+<br><img src="https://github.com/joyhughes/joyhughes.github.io/blob/main/request_response.png" height = 400><br>
```
Comment (rudi-cilibrasi) [inline Lux/README.md:24]: recommend dropping github.com part of URI so you can move this website off github if you ever want to more easily

Comment (rudi-cilibrasi) [reply Lux/README.md:24]: i think it's just `/joyhughes/joyhughes.github.io/blob/...`

Comment (joyhughes) [reply Lux/README.md:24]: Didn't see this before I merged. Where do you recommend putting images going forward?

Comment (rudi-cilibrasi) [reply Lux/README.md:24]: The image link you mention is on github.com. I can tell by reading the URI. So you don't need that hostname as part of that URI I believe nor do you need the https part.

Comment (joyhughes) [reply Lux/README.md:24]: Got your response on Discord to use relative URI

**Pull request merged**